SHTSensor.cpp

```diff
@@ -73,14 +73,9 @@ bool SHTI2cSensor::readFromI2c(uint8_t i2cAddress,
 
   Wire.requestFrom(i2cAddress, dataLength);
 
-  // there should be no reason for this to not be ready, since we're using clock
-  // stretching mode, but just in case we'll try a few times
-  uint8_t tries = 1;
-  while (Wire.available() < dataLength) {
-    delay(1);
-    if (tries++ >= MAX_I2C_READ_TRIES) {
```

Comment:
Looks good overall, but MAX_I2C_READ_TRIES should be removed, too, since it's not needed anymore